### DISCUSSION TITLE
CouchDB API

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#9163] [Help] [answered]
Is it possible to access the CouchDB via it's API in a self-hosted setup? Or is this only possible via Fauxton?

**Comment by melohagan** [comment, marked as answer]
Hey @ecxs 

If you can access the CouchDB instance via Fauxton then you should also be able to use the same config to with the API: https://docs.couchdb.org/en/stable/api/basics.html 

You'll want to be very careful when doing this however as you could potentially brick your apps.
